### PR TITLE
fix(docs): doc correction in plugin script and plugin script groovy

### PR DESCRIPTION
**Pull request by shrutimantri**

### CODE REVIEW

File: plugin-script/src/main/java/io/kestra/core/tasks/scripts/AbstractBash.java [hunk 1]
```diff
@@ -61,14 +61,14 @@ abstract public class AbstractBash extends Task {
 
     @Builder.Default
     @Schema(
-        title = "Interpreter args used"
+        title = "Interpreter arguments to be used."
```
Comment (shrutimantri) [inline plugin-script/src/main/java/io/kestra/core/tasks/scripts/AbstractBash.java:64]: Can someone please look at line 40 of this file, and confirm if that statement is correct?
I see that the default there is `RunnerType.PROCESS` as per line 45: `protected RunnerType runner = RunnerType.PROCESS`.

But then, here I see it is DOCKER: https://github.com/shrutimantri/plugin-scripts/blob/master/plugin-script/src/main/java/io/kestra/plugin/scripts/exec/AbstractExecScript.java#L33

Comment (anna-geller) [reply plugin-script/src/main/java/io/kestra/core/tasks/scripts/AbstractBash.java:64]: default is DOCKER

PROCESS was default very early on but it changed. you can try using any of those and you should see that by default it runs in Docker

File: plugin-script-groovy/src/main/java/io/kestra/plugin/scripts/groovy/FileTransform.java [hunk 2]
```diff
@@ -40,9 +40,9 @@
             }
         ),
         @Example(
-            title = "Transform with file from json string",
+            title = "Transform a json string to a file.",
```
Comment (anna-geller) [inline plugin-script-groovy/src/main/java/io/kestra/plugin/scripts/groovy/FileTransform.java:43]:
```suggestion
            title = "Transform a JSON string to a file.",
```

File: plugin-script/src/main/java/io/kestra/core/tasks/scripts/AbstractBash.java (hunk 1 above)
Comment (shrutimantri) [reply plugin-script/src/main/java/io/kestra/core/tasks/scripts/AbstractBash.java:64]: Is Line 45 correct then?

File: plugin-script/src/main/java/io/kestra/core/tasks/scripts/AbstractBash.java [hunk 3]
```diff
@@ -77,40 +77,40 @@ abstract public class AbstractBash extends Task {
...
         deprecated = true
     )
     @PluginProperty
     @Deprecated
     protected List<String> outputsFiles;
 
     @Schema(
-        title = "Output file list that will be uploaded to internal storage",
+        title = "Output file list that will be uploaded to Kestra's internal storage.",
         description = "List of key that will generate temporary files.\n" +
```
Comment (anna-geller) [inline plugin-script/src/main/java/io/kestra/core/tasks/scripts/AbstractBash.java:99]:
```suggestion
        description = "List of keys that will generate temporary files.\n" +
```

File: plugin-script/src/main/java/io/kestra/core/tasks/scripts/AbstractBash.java [hunk 4]
```diff
@@ -77,40 +77,40 @@ abstract public class AbstractBash extends Task {
...
-            "On the command, just can use with special variable named `outputFiles.key`.\n" +
-            "If you add a files with `[\"first\"]`, you can use the special vars `echo 1 >> {[ outputFiles.first }}`" +
-            " and you used on others tasks using `{{ outputs.taskId.outputFiles.first }}`"
+            "On the command, this can be used with special variable named `outputFiles.key`.\n" +
```
Comment (anna-geller) [inline plugin-script/src/main/java/io/kestra/core/tasks/scripts/AbstractBash.java:100]:
```suggestion
            "This property can be used with a special variable named `outputFiles.key`.\n" +
```

File: plugin-script/src/main/java/io/kestra/core/tasks/scripts/AbstractBash.java [hunk 5]
```diff
@@ -77,40 +77,40 @@ abstract public class AbstractBash extends Task {
...
-        title = "List of output directories that will be uploaded to internal storage",
+        title = "List of output directories that will be uploaded to Kestra's internal storage.",
         description = "List of keys that will generate temporary directories.\n" +
             "On the command, just can use with special variable named `outputDirs.key`.\n" +
-            "If you add a files with `[\"myDir\"]`, you can use the special vars `echo 1 >> {[ outputDirs.myDir }}/file1.txt` " +
-            "and `echo 2 >> {[ outputDirs.myDir }}/file2.txt` and both files will be uploaded to internal storage." +
-            " Then you can used them on others tasks using `{{ outputs.taskId.outputFiles['myDir/file1.txt'] }}`"
+            "If you add a file with `[\"myDir\"]`, you can use the special var `echo 1 >> {[ outputDirs.myDir }}/file1.txt` " +
+            "and `echo 2 >> {[ outputDirs.myDir }}/file2.txt`, and both the files will be uploaded to Kestra's internal storage. " +
+            "You can reference them on other tasks using `{{ outputs.taskId.outputFiles['myDir/file1.txt'] }}`."
```
Comment (anna-geller) [inline plugin-script/src/main/java/io/kestra/core/tasks/scripts/AbstractBash.java:113]:
```suggestion
            "You can reference them in other tasks using `{{ outputs.taskId.outputFiles['myDir/file1.txt'] }}`."
```

File: plugin-script/src/main/java/io/kestra/plugin/scripts/exec/scripts/models/DockerOptions.java [hunk 6]
```diff
@@ -39,27 +39,27 @@ public class DockerOptions {
...
+        title = "Docker image to use."
     )
     @PluginProperty(dynamic = true)
     @NotNull
     @NotEmpty
     protected String image;
 
     @Schema(
-        title = "User within the container"
+        title = "User within the docker container."
```
Comment (anna-geller) [inline plugin-script/src/main/java/io/kestra/plugin/scripts/exec/scripts/models/DockerOptions.java:50]:
```suggestion
        title = "User in the Docker container."
```

File: plugin-script/src/main/java/io/kestra/plugin/scripts/exec/scripts/models/DockerOptions.java [hunk 7]
```diff
@@ -113,14 +113,14 @@ public class DockerOptions {
...
-        description = "The size must be greater than 0. If omitted the system uses 64MB."
+        description = "The size must be greater than 0. If omitted, the system uses 64MB."
     )
     @PluginProperty(dynamic = true)
     private String shmSize;
 
     @Introspected
     @Schema(
-        title = "The PullPolicy for a container and the tag of the image affect when docker attempts to pull (download) the specified image."
+        title = "The PullPolicy for a container and the tag of the image affects when docker attempts to pull (download) the specified image."
```
Comment (anna-geller) [inline plugin-script/src/main/java/io/kestra/plugin/scripts/exec/scripts/models/DockerOptions.java:123]:
```suggestion
        title = "The image pull policy for a container image and the tag of the image, which affect when Docker attempts to pull (download) the specified image."
```

File: plugin-script/src/main/java/io/kestra/core/tasks/scripts/AbstractBash.java (hunk 1 above)
Comment (loicmathieu) [reply plugin-script/src/main/java/io/kestra/core/tasks/scripts/AbstractBash.java:64]: This class is the parent of the old deprectated script tasks that uses the PROCESS runner by default.

I suggest changing the documentation title to just `The script runner` with no mention as default as defaults are automatically documented.

**Review by anna-geller — APPROVED** [review]

**Pull request merged**